composer.json

```diff
@@ -14,7 +14,7 @@
         "php": "^5.5 || ^7.0",
         "php-http/httplug": "^1.0",
         "php-http/discovery": "^1.0",
-        "cakephp/cakephp": "^3.1"
+        "cakephp/cakephp": "^3.4"
```

Comment:
Use 3.4.5 and the test will pass. 

Versions before 3.4.5 had wrong dependencies.